server/index.js

```diff
@@ -1,24 +1,22 @@
-require('newrelic');
+var nr = require('newrelic');
```

Comment:
What does this do?

Reply:
Right now I'm not doing anything with it. But I believe nr comes with methods I can add later.